src/browser/of_events.ts

```diff
@@ -42,6 +42,15 @@ class OFEvents extends EventEmitter {
 
         return super.emit(route, ...data);
     }
+
+    public SUBSCRIBER: StringMap = {
```

Comment:
So for `static`, you would also `import` the `OfEvents` constructor itself which would have the static members already so
````js
import {default as whatever, OfEvents} from '../of_events';

ofEvents.SUBSCRIBER.ADDED // ... 
````

But you have convinced me that its fine as is 😄